Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.9-buster as base
+
+RUN pip install "poetry==1.1.4"
+COPY poetry.lock pyproject.toml .
+RUN poetry install
+
+COPY ./todo_app /todo_app/
```

Comment:
I'd be inclined to match the local structure slightly more to make sure things keep working in future, so e.g. setting a `WORKDIR` to `/app` which can hold our `poetry.lock`/`pyproject.toml` files and then having the `todo_app` folder below that

As it is, we have those poetry files at the very root folder of our system, which could be a little confusing